packages/sdk/pnpm-lock.yaml

```diff
@@ -2045,7 +2045,7 @@ packages:
     dependencies:
       '@azure/core-http': 3.0.4
       '@azure/msal-node': 1.14.6
-      axios: 1.6.2
+      axios: 1.6.7
```

Comment:
What's the change?

Reply:
Before the change: " [no entry for '/axios/1.6.2' in pnpm-lock.yaml](https://github.com/OfficeDev/TeamsFx/pull/10920/commits/342651a049c0a461aff9c91f54d7e0acd68009bf) ".
![image](https://github.com/OfficeDev/TeamsFx/assets/139844715/b69589fa-f0c9-4f95-89f1-5d2a07de25c0)

Reply:
This is resolved in [fix: fix sdk deps (#10921) · OfficeDev/TeamsFx@5f938f3 (github.com)](https://github.com/OfficeDev/TeamsFx/commit/5f938f3d045b9c9658ab7c7a90f50f29139ca1d3)